spec/models/post_spec.rb

```diff
@@ -1311,4 +1311,18 @@ def post_with_body(body, user = nil)
     end
   end
 
+  describe '#indexable?' do
+    it "should return true for regular posts and whispers" do
+      results = {}
+
+      Post.types.each do |name, post_type|
+        post = Fabricate(:post, post_type: post_type)
+        results[name] = post.indexable?
+      end
+
+      expect(results.reject { |_, v| v == true }.keys)
+        .to contain_exactly(:moderator_action, :small_action)
```

Comment:
This is intentional so the spec will fail when someone adds a new post action type in the future.